bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VBFToXToHH_UL/VBFToBulkGravitonToHH_M/VBFToBulkGravitonToHH_M_extramodels.dat

```diff
@@ -1,2 +1,2 @@
 ## Models for searches of diboson resonances
-dibosonResonanceModel_toVBF.tar.gz
+dibosonResonanceModel_toVBF_onefolder.tar.gz
```

Comment:
Pointing to an updated model which contains a single folder for necessary directory structure when importing models in madgraph cards.